app/assets/stylesheets/checkin.scss

```diff
@@ -0,0 +1,7 @@
+.disable-field-set {
+  background: gray;
+}
+
+.disable-field-set h5 {
+  color: #bbb;
```

Comment:
Color literals like `#bbb` should only be used in variable declarations; they should be referred to via variable everywhere else.